src/templates/antora/template.ts

```diff
@@ -16,26 +16,43 @@ interface NavFile {
 
 type PageOutput = AntoraTemplate.PageOutput;
 
-const BASE_PATH = process.env.BASE_PATH || '/_data/antora';
-
-// correlates to tinymce-docs antora path
-const AntoraNavBaseDir = 'apis/';
-
-const navToAdoc = (navyml: NavFile[]): string => {
-  // Api index page
-  const indexPage = navyml[0];
-  let adoc = '* ' + indexPage.title + '\n';
-
-  // generate API namespaces
-  indexPage.pages.forEach((namespace) => {
-    // main namespace level navigation (namespace index)
-    adoc += '** ' + namespace.path + '\n';
-    namespace.pages.forEach((page) => {
-      // namespace level pages
-      adoc += '*** xref:' + AntoraNavBaseDir + page.path + '.adoc' + '[' + page.title + ']\n';
+const generateXref = (basePath: string, filename: string, title: string) =>
+  'xref:' + basePath + '/' + filename + '[' + title + ']';
+
+const navLine = (name: string, level: number): string =>
+  '*'.repeat(level) + ' ' + name + '\n';
+
+const navToAdoc = (indexPage: NavFile, structure: string): string => {
+  // Api index nav page top level
+  let adoc = navLine(indexPage.title, 1);
+  if (indexPage.pages) {
+    indexPage.pages.forEach((namespace) => {
+      const namespaceLine = structure === 'legacy' ? generateXref('api/' + namespace.path, namespace.path + '_nav.adoc', namespace.title) : namespace.title;
+      adoc += navLine(namespaceLine, 2);
+      if (namespace.pages) {
+        namespace.pages.forEach((pageFile) => {
+          const pagePath = structure === 'legacy' ? 'api/' + namespace.path : 'apis';
+          adoc += navLine(generateXref(pagePath, pageFile.path + '.adoc', pageFile.title), 3);
+        });
+      }
     });
-  });
+  }
+  return adoc;
+};
 
+const indexToAdoc = (namespace: NavFile): string => {
+  let adoc = '= ' + namespace.title + '\n\n';
```

Comment:
This needs the `description` and `keywords` added to it like it did in the old `tinymcenext` template: https://github.com/tinymce/moxiedoc/blob/39b04d910f4e85b9026406d26276286d9a2dd749/src/templates/tinymcenext/member.handlebars#L4-L5

Reply:
This probably should be using the `member.handlebars` template to generate the initial file like we do for other files.

Reply:
Done